nRFCloud/device-simulator-v2
# feat: Add alerts and logs

### Summary
We are now supporting appId LOG and ALERT on nRFCloud and for testing purposes it is needed to have these sent out from a simulated device.

### Testing
1. Checkout this branch
2. Log into dev.nrfcloud.com
3. `yarn build`
4. `node dist/cli.js -k <API-key> -h https://api.dev.nrfcloud.com -a -d <DEVICE-ID> -s log,alert`
5. You can see the alerts every ~2 minutes from polling (or live in the terminal card). Logs you should be able to see real time in the device's logs card.

### Notes
* I have added the "control" section to the shadow when you change the value in the shadow, but it won't actually have an effect on the device sending the data to those restrictions.
* For an unknown reason, you always need the -a flag for it to work.
